crates/chain/src/indexer/spk_txout.rs

```diff
@@ -334,4 +340,23 @@ impl<I: Clone + Ord + core::fmt::Debug> SpkTxOutIndex<I> {
             .any(|output| self.spk_indices.contains_key(&output.script_pubkey));
         input_matches || output_matches
     }
+
+    /// Find relevant script pubkeys associated with a transaction for tracking and validation.
+    ///
+    /// Returns a set of script pubkeys from [`SpkTxOutIndex`] that are pertinent to the outputs and
```

Comment:
`pertinent` is synonymous with `relevant` right? Can we just use `relevant` to be consistent and it's a much more common term.